code/global.dm

```diff
@@ -246,6 +246,142 @@ var/global
 	"Heterochromia: Left" = "hetcroL",
 	"Heterochromia: Right" = "hetcroR")
 
+	list/hair_styles = list("Afro",
```

Comment:
ugh this should all just be datumized 😿

Reply:
Would that mean making a everything into classes? For example, a customization_class with children hair_styles, beard_styles, and moustache_styles. The relevant checks then turning into istype rather than checking for list membership?

Reply:
yeah, but not necessary for this PR at all